SpriteBuilder/ccBuilder/ResourceManagerUtil.m

```diff
@@ -71,9 +73,12 @@ + (void) setTitle:(NSString *)str forPopup:(NSPopUpButton *)popup
     [self setTitle:str forPopup:popup forceMarker:NO];
 }
 
-+ (void) addDirectory: (RMDirectory*) dir ToMenu: (NSMenu*) menu target:(id)target resType:(int) resType allowSpriteFrames:(BOOL) allowSpriteFrames
++ (NSInteger) addDirectory: (RMDirectory*) dir ToMenu: (NSMenu*) menu target:(id)target resType:(int) resType allowSpriteFrames:(BOOL) allowSpriteFrames
 {
     NSArray* arr = [dir resourcesForType:resType];
+    NSInteger count = 0; // Valid Assets Added
+    
+    [menu setDelegate:(id)self];
```

Comment:
I had the delegate set originally in `populateResourcePop` however when the delegate gets called it ends up having no `RMResource` object set in `representedObject`however fine when set in `addDirectory` didn't look into it too much after that.